### ISSUE TITLE
Remove `No Ads` entry from descriptions

### ISSUE THREAD

**Issue by licaon-kter** [organicmaps/organicmaps#6668]
ref: https://github.com/organicmaps/organicmaps/pull/6523

ref: https://gitlab.com/fdroid/fdroiddata/-/merge_requests/14082

**Comment by pastk** [comment]
FYI @rtsisyk @biodranik

**Comment by biodranik** [comment]
Let's make it clear: we do not consider the "Details on Kayak" button as an ad. Because:
- Nobody pays us to display the "Details on Kayak" button.
- It is displayed only for some selected hotels in the right context (when a user is interested to see more details about a hotel)
- It looks like any other hotel/POI metadata (website, FB link, Twitter link, phone, email, etc.)
- It is not intrusive or annoying to our users like banners in other apps on FDroid marked with "ad" anti-feature.
- It is completely optional to click or not to click. The user decides that.
- Clicking on the link displays an explicit disclaimer dialog where the user learns about our affiliate fee in case any booking is done, and can press "Cancel" if necessary.
- This button itself brings a lot of value for users, it shows photos, and reviews, and allows them to book the hotel if necessary.
- The expected affiliate income from these links is unlikely to compensate even our time spent discussing this issue. But it is useful to discover different opinions on that topic.

If the fact that we get an affiliate fee from these links after an explicit disclaimer means for someone that it is an ad, then our "Donate" buttons should be also considered as ads. Because we also get the "affiliate fee" when users are paying using our donate links. And we even don't show an explicit disclaimer that we'll get money for project development if users donate to us! Furthermore, should any app that displays the "Donate" button/banner/dialog get the "Has an ad anti-feature" label on FDroid?

**Comment by licaon-kter** [comment]
I find it a stretch, a "whataboutism" to put Donate in the same pot as referral links. Pls don't.

**Comment by biodranik** [comment]
I agree, let's focus on the right solution. I was emotionally referring to [Justin's words](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/14082#note_1666056780):
> Are you getting money by presenting a link where you can buy something? It's advertising.

**Comment by pastk** [comment]
I think the easiest option is just removing the link from the F-Droid build.


Still, it won't put an end to the "ad vs not-an-ad" debate. Clearly the matter is quite subjective and also its not practical to defend a "OM's referral links are not ads" point every now and then to new users.

So I suggest to clarify the "No ads" claim, e.g. change it to something like "No annoying ads". And further in the text (when applicable, e.g. in store's description, on the web site, etc.) describe OM's policy re referral links and how they provide value to the users while not being intrusive, being explicitly disclosed etc.

**Comment by rtsisyk** [comment]
https://github.com/organicmaps/organicmaps/pull/6722

**Comment by rtsisyk** [comment]
What is the point of spreading discussing into one more ticket? Here is the original thread: https://github.com/organicmaps/organicmaps/pull/6523

**Comment by matkoniecz** [comment]
> Nobody pays us to display the "Details on Kayak" button.

It seems to be affiliate link? Yes, payment is conditional on people booking with it but in the end you are paid for it.

> It is completely optional to click or not to click. The user decides that.

The same goes for all ads (except the worst cases of malware)

> If the fact that we get an affiliate fee from these links after an explicit disclaimer means for someone that it is an ad, then our "Donate" buttons should be also considered as ads. Because we also get the "affiliate fee" when users are paying using our donate links. And we even don't show an explicit disclaimer that we'll get money for project development if users donate to us! Furthermore, should any app that displays the "Donate" button/banner/dialog get the "Has an ad anti-feature" label on FDroid?

Donate button lack combination of promoting service in exchange for money.

(note also that something can be an ad without being paid for it, but that is a separate can of worms)

**Comment by Kasi78** [comment]
[They got anti feature 😂](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/14082/diffs?commit_id=cdf6baa74e0f222ebb210ae63ca4d51a4b3807a8)

https://github.com/organicmaps/organicmaps/issues/7119 

How pitiful 😂

**Comment by pastk** [comment]
@rtsisyk why locked?
